src/connectors/atomService.ts

```diff
@@ -29,27 +49,114 @@ interface DeleteManyInput {
   whereIn?: [string, string[]]
 }
 
+interface CountInput {
+  table: TableName
+  where: Record<string, any>
+}
+
 /**
  * This object is a container for data loaders or system wide services.
  */
 export class AtomService extends DataSource {
   es: typeof es
+  aws: typeof aws
   knex: Knex
 
+  circleIdLoader: DataLoader<string, Item>
+  draftIdLoader: DataLoader<string, Item>
+  userIdLoader: DataLoader<string, Item>
+
   constructor() {
     super()
     this.es = es
+    this.aws = aws
     this.knex = knex
+
+    this.circleIdLoader = this.initLoader({ table: 'circle', mode: 'id' })
```

Comment:
Do you think it's better to implement the dataloaders here, or should we allow resolvers to pass in table name and create their own dataloader (for example with `initLoader`)? cc: @robertu7

Reply:
😉 Have no strong preference to it, but I guess we'll have redundant loaders easily. For example, we need `article`, `comment`, `user` loaders when resolving `User`, and also need `comment`, `draft` and `user` loaders when resolving `Article`.